core/arch/riscv64/riscv64.asm

```diff
@@ -40,6 +40,16 @@ START_FILE
 # error Non-Linux is not supported
 #endif
 
+/* sizeof(priv_mcontext_t) rounded up to a multiple of 16 */
+#define PRIV_MCONTEXT_SIZE 0x410
```

Comment:
Please add to the comment how this computes to be 0x410 (1040) bytes.
We have 32 int regs, 32 float regs, 1 fcsr, 1 pc, all 8 bytes (I think), that's 528 bytes?

Or do we reserve space for the MCXT_NUM_SIMD_SLOTS simd regs even though we don't save them below? If so, please add to comment.

Reply:
Yes, we do have some reserved space for SIMD, and it won't compile if removed. I added that comment in [cd9d282](https://github.com/DynamoRIO/dynamorio/pull/6064/commits/cd9d28256b90f4b95f30fea7e1ba92d4d818df75).

Reply:
Seems I computed this wrong somehow, 528 + 8*16 = 656, which is 0x290.

Reply:
Okay, thanks for finding and correcting.